tests/conftest.py

```diff
@@ -18,6 +18,10 @@
     "9.0.2022090601",
 ]
 
+MOCKED_GCP_IMAGE_LIST = [
+    {"status": "READY"},
```

Comment:
Maybe add another line with status = DEPRECATED to see that the data is properly filtered by cli.gcp_images

Reply:
Makes sense - added an entry :)